.github/workflows/tag_image_push.yml

```
@@ -57,18 +67,19 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: hazelcast/hazelcast
-          short-description: Hazelcast IMDG Docker Image
+          repository: ${DOCKER_ORG}/hazelcast
```

I don't think the actions can receive the environment variables in shell style. I believe the line should be as follows
```suggestion
          repository: ${{ env.DOCKER_ORG }}/hazelcast
```

---

Actually, it is working but you are right we should follow convention 👍